rtscomponentkit/src/main/java/io/dolby/rtscomponentkit/domain/StreamingData.kt

```diff
@@ -18,19 +24,51 @@ data class ConnectOptions(
     companion object {
         fun from(
             useDevEnv: Boolean,
+            serverEnv: String,
             forcePlayOutDelay: Boolean,
             disableAudio: Boolean,
             rtcLogs: Boolean,
             primaryVideoQuality: String,
             videoJitterMinimumDelayMs: Int
-        ): ConnectOptions =
-            ConnectOptions(
+        ): ConnectOptions {
+            val env = if (serverEnv.isEmpty()) {
+                if (useDevEnv) {
+                    ENV.DEV
+                } else {
+                    ENV.safeValueOf(serverEnv)
+                }
+            } else {
+                ENV.safeValueOf(serverEnv)
+            }
+            return ConnectOptions(
                 useDevEnv = useDevEnv,
+                serverEnv = env,
                 forcePlayOutDelay = forcePlayOutDelay,
                 disableAudio = disableAudio,
                 rtcLogs = rtcLogs,
                 primaryVideoQuality = VideoQuality.valueToQuality(primaryVideoQuality),
                 videoJitterMinimumDelayMs = videoJitterMinimumDelayMs
             )
+        }
+    }
+}
+
+enum class ENV {
```

Comment:
Kotlin naming convention would be to name this like any other class, while its values are all caps

Could be clearer on the name since Env will duplicate other java classes. ApiEnv or DirectorEnv or MillicastEnv or MediaEnv?